stdlib/_interpreters.pyi

```diff
@@ -0,0 +1,48 @@
+import types
+from collections.abc import Callable, Mapping
+from typing import Final, Literal, SupportsIndex, final
+from typing_extensions import TypeAlias
+
+_Configs: TypeAlias = Literal["default", "isolated", "legacy", "empty"]
```

Comment:
It appears to also allow the empty string.

Reply:
Good catch: https://github.com/python/typeshed/pull/12230/commits/ef338069a1f2094dc4e8ecbffc63305c696ad5ed